api_tests/tests/reasonable_adjustment_flag/test_error_cases.py

```diff
@@ -117,8 +117,8 @@ def test_missing_nhsd_session_urid_header(self):
         # Given
         expected_status_code = 400
         expected_response = {
-            "error": "invalid header",
-            "error_description": "nhsd-session-urid is missing or invalid"
+            "error": "invalid role",
+            "error_description": "NHSD-Session-URID is invalid"
```

Comment:
I think the standard for error messages is to be all lower case, so might need to change the policy that throws this error message.